packages/amazonq/test/unit/codewhisperer/util/runtimeLanguageContext.test.ts

```diff
@@ -288,7 +287,6 @@ describe('runtimeLanguageContext', function () {
             ['jsx', 'jsx'],
             ['kotlin', 'kt'],
             ['php', 'php'],
-            ['plaintext', 'txt'],
```

Comment:
is tested here
https://github.com/aws/aws-toolkit-vscode/blob/master/packages/amazonq/test/unit/codewhisperer/util/runtimeLanguageContext.test.ts#L106